public/assets/css/main.css

```diff
@@ -19,6 +19,11 @@ h3 {
 
 p {
 	font-weight: 300;
+	margin: 3em;
+}
+
+a {
+	color: #FFF;
```

Comment:
Prefer `inherit` (unless `#FFF` is an actual requirement and not just a repetition of `body`'s `color`.